chatapi_ros/README.md

```diff
@@ -0,0 +1,90 @@
+# chatapi_ros
+The purpose of this repository is to make existing chat APIs usable with ROS.
+
+## Setting up environment
+Please install following packages.
+```
+sudo pip install google-api-python-client
```

Comment:
do we need this in additino to ? https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/2b0226ec32e76176eff179f61f7dd77461e3fa4e/ros_speech_recognition/package.xml#L23 (`pip install SpeechRecognition`), if so please update https://github.com/jsk-ros-pkg/jsk_3rdparty/tree/master/ros_speech_recognition